### DISCUSSION TITLE
LDAP Issues

### DISCUSSION THREAD

**Discussion by twinsmom71** [snipe/snipe-it#9335] [Q&A]
Hello, having an issue with my LDAP sync.  It will not pull in the Employee No field.  We use attribute employeeID for our Employee No.  All other fields seem to be importing just fine.  I have attached my LDAP settings, any ideas what I am doing wrong?
![image](https://user-images.githubusercontent.com/79875860/112156569-bdd69000-8bbc-11eb-9e5c-8bf214fd0d5c.png)

**Comment by twinsmom71** [comment]
So interesting, if I change the Employee No to an attribute with TEXT, LDAP Sync works fine. When changes to an attribute that contains numbers only, it leaves it all blank. Is this a limitation for the Employee No field?

**Comment by snipe** [comment]
>  if I change the Employee No to an attribute with TEXT

This seems more like an issue than a discussion? The `employee_num` field is a text string in the database, so I don't see any reason why it should care whether it's numbers or text.
